_config/debugbar.yml

```diff
@@ -51,3 +51,5 @@ SilverStripe\Admin\LeftAndMain:
 SilverStripe\Control\Controller:
   extensions:
     - LeKoala\DebugBar\Extension\ControllerExtension
+LeKoala\DebugBar\DebugBar:
+  show_namespaces: false
```

Comment:
Ah, overlooked that block.

I'm not a fan of long config names, I rather add a comment explaining what it does than writing complete works of Shakespeare as config name.